src/components/Airdrop/Airdrop/index.tsx

```diff
@@ -0,0 +1,117 @@
+import { AVALANCHE_MAINNET, AirdropType, Chain, Token, TokenAmount } from '@pangolindex/sdk';
+import { formatUnits } from 'ethers/lib/utils';
+import React, { useCallback, useEffect, useState } from 'react';
+import { Box, Button, Loader, Text, Tooltip } from 'src/components';
+import { usePangolinWeb3 } from 'src/hooks';
+import { useClaimAirdrop, useMerkledropClaimedAmounts, useMerkledropProof } from 'src/state/airdrop/hooks';
+import { getChainByNumber } from 'src/utils';
+import Title from '../Title';
+import { Wrapper } from '../styleds';
+import AlreadyClaimed from './Claimed';
+import ConfirmDrawer from './ConfirmDrawer';
+import NotEligible from './NotEligible';
+import ChangeChain from './changeChain';
+import NotConnected from './notConnected';
+
+interface Props {
+  /** Airdrop Contract address */
+  contractAddress: string;
+  /** Airdrop contract type */
+  type: AirdropType;
+  /* Title of airdrop*/
+  title?: string;
+  /** Token to be airdropped */
+  token: Token;
+  /** Token Logo */
+  logo: string;
+  /** The airdropped chain */
+  chain?: Chain;
+  /** Function to executed  on complete*/
+  onComplete?: (airdropType) => void;
+}
+
+export default function Airdrop({ contractAddress, type, title, logo, token, chain, onComplete }: Props) {
```

Comment:
1. Lets keep entire Airdrop code same as what we have in interface right now. Because for now no one is going to use Airdrop component except Interface so we dont need to add extract configuration for that.
2. Lets keep same folder structure as interface
3. run yarn lint and tsc to make sure there is no warning and errors.